.github/workflows/java_sdk_logging.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Github action job to test core java library features on
+# downstream client libraries before they are released.
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    paths:
+    - java-sdk-logging/**
+
+name: java-sdk-logging-ci
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: temurin
+    - name: Install parent module
+      run: |
+        mvn install -B -ntp -pl gapic-generator-java-pom-parent
+    - name: Unit Tests
+      working-directory: java-sdk-logging
+      run: |
+        mvn test -B -ntp -Dcheckstyle.skip -Dfmt.skip
+  module-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: 11
+        distribution: temurin
+    - name: Install parent module
+      run: |
+        mvn install -B -ntp -pl gapic-generator-java-pom-parent
+    - name: Java Linter
```

Comment:
In addition to linter, can we add clirr check and enforcer check?

Reply:
Sure, I'll add them in a follow up PR.